scripts/measures/archived/scripts/get-stratification.js

```diff
@@ -67,23 +107,21 @@ Promise.all(
   .then(docs => {
     return _.compact(docs.map(doc => {
       const measure = doc.QualityMeasureDocument;
-      const emeasureid = measure.subjectOf[0].measureAttribute[0].value[0].$.value;
+      const measureId = measure.subjectOf[0].measureAttribute[0].value[0].$.value;
       const strata = extractStrata(measure);
       if (_.isEmpty(strata)) {
         return;
       }
       const version = measure.versionNumber[0].$.value.split('.')[0];
-      const eMeasureId = `CMS${emeasureid}v${version}`;
+      const eMeasureId = `CMS${measureId}v${version}`;
       return {
         eMeasureId,
-        strata
+        mapping: strata
```

Comment:
WDYT about `strataMaps: strataMaps` or something similar?

Reply:
I like it